lib/commands/devices/index.ts

```diff
@@ -116,7 +123,9 @@ export default class DevicesCmd extends Command {
 
 		if (options.json) {
 			const { pickAndRename } = await import('../../utils/helpers');
-			const mapped = devices.map((device) => pickAndRename(device, fields));
+			const mapped = devices.map((device) =>
+				pickAndRename(device, [...fields, 'device_tag']),
```

Comment:
@thgreasi You probably have more context than I do on this, should the, now being expanded, device_tag have the pickAndRename applied to it? AFAIK this only applies in case of something being formatted with 'a => b' and we probably wouldn't want to change it if this is the tag value of the client?

Reply:
@otaviojacobi I think that we only use `pickAndRename` in this file for consistency reasons and that in this file using `pick` or `pickAndRename` has the same outcome since there is no `=>` used in the defined fields.